_chapters/chapter_01.md

```diff
@@ -114,22 +130,22 @@ layout: rails_tutorial
 {% endsteps %}
 
 {% protip %}
-  Don't have a favorite text editor for programming? We recommend [Atom](https://atom.io/).
+  It's easiest to edit these files using a text editor created just for programming. If you don't already have a favorite text editor for programming? We recommend [Atom](https://atom.io/).
```

Comment:
Nice! I hard time wording this one. The last couple of sentences are kinda funny though. Maybe you forgot to finish your thought?

>If you don't already have a favorite text editor for programming? We recommend [Atom](https://atom.io/).